integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java

```diff
@@ -129,9 +132,16 @@ public void healthCheckTest() throws Exception {
     assertEquals(ResponseStatusCode.INTERNAL_ERROR, response.getStatusCode());
 
     tempDataFile.delete();
-    Uninterruptibles.sleepUninterruptibly(3, TimeUnit.SECONDS);
-    nodes  = coordinators.get(0).getClusterManager()
-        .getServerList(Sets.newHashSet(Constants.SHUFFLE_SERVER_VERSION));
+    int i = 0;
+    do {
```

Comment:
can the retry strategy be replaced with CoordinatorTestUtils.waitForRegister?

Reply:
We fetch the ServerNode objects, not only the number of the servers. So it cannot be replaced.